### PR TITLE
feat: Update konnector messages

### PR DESCRIPTION
**Pull request by fffflo**
new trads regarding connectors error

### CODE REVIEW

File: packages/cozy-harvest-lib/src/locales/en.json [hunk 1]
```diff
@@ -77,20 +77,20 @@
...
         "description": "%{name} seems to have updated its Terms Of Service. Please check that the service is up to date. It this error still occurs, please contact us at <a href=\"mailto:contact@cozycloud.cc\">contact@cozycloud.cc</a>."
       },
       "UNKNOWN_ERROR": {
         "title": "Connection error",
         "description": "An unknown error has occurred."
       },
       "USER_ACTION_NEEDED": {
-        "title": "Action needed on the provider",
-        "description": "It seems that the [%{name}](%{link}) website requires you to log in and to do a specific action. Please rerun the connector once you have settled the issue on the website."
+        "title": "Action needed on provider website",
```
Comment (ptbrowne) [inline packages/cozy-harvest-lib/src/locales/en.json]:
```suggestion
        "title": "Action needed on the provider's website",
```

I am not sure but it sounds less robotic

Comment (fffflo) [reply packages/cozy-harvest-lib/src/locales/en.json]: sounds better yes

File: packages/cozy-harvest-lib/src/locales/en.json [hunk 2]
```diff
@@ -101,23 +101,23 @@
...
       "USER_ACTION_NEEDED.TWOFA_EXPIRED": {
-        "title": "Relaunch the connexion to the service to fetch your data",
+        "title": "Authentication renewal required",
         "description": "The last connexion to the service failed; please launch it again. You may have to provide a validation code."
       },
       "USER_ACTION_NEEDED.WRONG_TWOFA_CODE": {
-        "title": "Wrong two-factore code provided",
-        "description": "The two-factor code provided is wrong, please start again."
+        "title": "Incorrect strong authentication code",
+        "description": "The second-factor code provided is incorrect, please start again."
```
Comment (ptbrowne) [inline packages/cozy-harvest-lib/src/locales/en.json]: I think two-factor was ok

Comment (fffflo) [reply packages/cozy-harvest-lib/src/locales/en.json]: both seem possible according to google

**Pull request merged**